### DISCUSSION TITLE
Image optimization

### DISCUSSION THREAD

**Discussion by jsbrain** [maizzle/framework#1286] [Q&A]
Hey @cossssmin,

I was wondering if there are any plans to add automatic image optimization to the framework or do you have any recommendation how one would plug this into the framework as of now?

**Comment by cossssmin** [comment]
You could use something like [sharp](https://github.com/lovell/sharp) in the [`afterBuild` event](https://maizzle.com/docs/events#afterbuild), applying it to all image files in-place.

No plans on integrating this into core but I'll have a guide written when I get some time 👍

**Reply by jsbrain** [reply]
That was exactly my first thought. Cool, will try to get something working and will report back, maybe we could turn this into a guide section or so.
